src/frontend/src/btc/derived/tokens.derived.ts

```diff
@@ -8,7 +8,7 @@ export const enabledBitcoinTokens: Readable<Token[]> = derived([testnets], ([$te
 	NETWORK_BITCOIN_ENABLED
 		? [
 				...(BTC_MAINNET_ENABLED ? [BTC_MAINNET_TOKEN] : []),
-				...($testnets ? [BTC_TESTNET_TOKEN] : [])
+				...($testnets ? [BTC_TESTNET_TOKEN, BTC_REGSTEST_TOKEN] : [])
```

Comment:
Shoudn't Regtest be enabled only for devs (only if running env is LOCAL)? Or the idea is that any user can decide to run the Regtest node locally if they want?

Reply:
@lmuntaner can you please take a look at Antonio's question?

Reply:
Yes, that's makes a lot of sense, I agree.

Reply:
Agreed, it's now added to the array only in the local env